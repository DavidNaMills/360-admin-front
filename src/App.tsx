import React from "react";
import classes from "./App.module.scss";

const App = (): JSX.Element => {
  console.log("hello!");
  return (
    <div className={classes.hello}>
      <p> hello!</p>
    </div>
  );
};

export default App;
