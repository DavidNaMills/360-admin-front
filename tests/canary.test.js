import React from "react";
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from "react-test-renderer";
import App from "../src/App";

describe("the canary cannot die", () => {
  it("doesnt kill the canary", () => {
    expect.assertions(1);
    expect(true).toBe(true);
  });

  it("ensures the ENV variables have been loaded", () => {
    expect.assertions(1);
    expect(process.env.ENV).toStrictEqual("tests");
  });

  it("takes a picture of the canary", () => {
    expect.assertions(1);
    const comp = renderer.create(<App />);
    const app = comp.toJSON();
    expect(app).toMatchInlineSnapshot(`
      <div
        className="hello"
      >
        <p>
           hello!
        </p>
      </div>
    `);
  });
});
