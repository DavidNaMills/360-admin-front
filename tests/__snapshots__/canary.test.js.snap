// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the canary cannot die takes a picture of the canary 1`] = `
<div>
  <p>
     hello!
  </p>
</div>
`;
